packages/rich-text/src/component/index.js

```diff
@@ -325,8 +325,32 @@ function RichText(
 			multilineTag,
 			preserveWhiteSpace,
 		} );
-		event.clipboardData.setData( 'text/plain', plainText );
-		event.clipboardData.setData( 'text/html', html );
+
+		const isLi = /<li>.*<\/li>/gi;
+		const foundLi = html.match( isLi );
+		if ( foundLi ) {
+			let liCopyString = html
+				.split( '<li>' )
+				.join( '' )
+				.split( '</li>' )
+				.filter( Boolean );
+			const liCopyStringLength = liCopyString.length - 1;
```

Comment:
there is no need of adding this variable here, we can get a length of it inside `map` operator

```
.map( ( el, I, {length} ) => {
  if ( el && i === 0 ) {
	  return '<ul><li>' + el + '</li>';
  } else if ( el && i === length - 1 ) {
	  return '<div style="clear:both"/><li>' + el + '</li></ul>';
  }
  return '<div style="clear:both"/><li>' + el + '</li>';
} );
```

Reply:
nice tip ) can you give me a link for tips like this , I first time see that I can get length in arguments of callback) where did you get this tip ?)

Reply:
uummm... this is just an object destructuring (an array is an object, and it is passed as the third parameter to the callback). 
Please check the documentation https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/map